### ISSUE TITLE
How to obtain user's attitude information and perspective information

### ISSUE THREAD

**Issue by LouHan-0325** [ValveSoftware/openvr#1439]
I am a novice at openVR and I want to obtain user's attitude and perspective information.Which module can be applied？

Thanks for any help.

**Comment by BOLL7708** [comment]
Are you talking about a BCI solution to read actual brain activity, or is it a misspelling of altitude and refering to a user's physical perspective, meaning their head pose? Just not sure what you are asking for here, and if it is the prior perhaps not the right forum for it? 😅

**Comment by LouHan-0325** [comment]
> Are you talking about a BCI solution to read actual brain activity, or is it a misspelling of altitude and refering to a user's physical perspective, meaning their head pose? Just not sure what you are asking for here, and if it is the prior perhaps not the right forum for it? 😅

I‘m sorry that perhaps it's a misspelling. I mean that I want to obtain user's head pose information and their viewpoint position.

**Comment by BOLL7708** [comment]
The head pose can be retrieved directly from the OpenVR API, do you have any specific needs when it comes to language etc? I've made data retrieval applications for years now, and the latest public [one](https://github.com/BOLL7708/OpenVR2WS) returns pose data as JSON via WebSockets, not sure if that's helpful. Asking for a module makes it sound like this is for integration in some project though?

**Comment by LouHan-0325** [comment]
Yes, I want to get the user's viewpoint position in order to transmit high resolution video tiles in user's FOV. So can I get head pose through [IVRCompositor::WaitGetPoses](https://github.com/ValveSoftware/openvr/wiki/IVRCompositor::WaitGetPoses)? or some fuction else?
